src/elm/Page/Login.elm

```diff
@@ -318,14 +319,24 @@ view env model =
                         |> B.setOnClick (Just HideInfoStep)
                         |> B.primary B.Primary_2
                     ]
-                , B.init "Les mer om AtBs nye reisetjenester her"
-                    |> B.setElement H.a
-                    |> B.setAttributes [ A.href "https://www.atb.no/vi-oppgraderer/" ]
-                    |> B.link
-                , B.init "For our English speaking travellers"
-                    |> B.setElement H.a
-                    |> B.setAttributes [ A.href "https://www.atb.no/ny-nettbutikk/key-words-and-phrases-article17509-2740.html" ]
-                    |> B.link
+                , case appInfo.newTravelServicesUrl of
+                    Just newTravelServicesUrl ->
+                        B.init "Les mer om våre nye reisetjenester her"
+                            |> B.setElement H.a
+                            |> B.setAttributes [ A.href newTravelServicesUrl ]
+                            |> B.link
+
+                    Nothing ->
+                        Html.Extra.nothing
+                , case appInfo.englishTranslationsUrl of
+                    Just englishTranslationsUrl ->
+                        B.init "For our English speaking travellers"
+                            |> B.setElement H.a
+                            |> B.setAttributes [ A.href englishTranslationsUrl ]
```

Comment:
Her kunne også brukt en attribute med lang som en for mer info til skjermlesere.

Reply:
God idé, la til `en` som lang-attributt.